src/core/blocks/math.hpp

```diff
@@ -83,48 +86,63 @@ struct BinaryBase : public Base {
     return ComposeError(errStream.str());
   }
 
+  void validateTypes(const CBTypeInfo &lhs, const CBType &rhs, CBTypeInfo &resultType) {
```

Comment:
this was long overdue.. but not sure why I never did it... for some reason prevented something. If it works cool though :)

Reply:
I noticed the broadcast operators were missing when trying to translate them for shaders. decided to add them 😄